docs/src/get-started/tutorials/understanding_hypers.py

```diff
@@ -0,0 +1,405 @@
+
+
+def shifted_cosine(r, cutoff, width):
+    """A shifted cosine switching function.
+
+    Parameters
+    ----------
+    r : float
+        distance between neighbouring atoms in Å
+    cutoff : float
+        cutoff distance in Å
+    width
+        width of the switching Å
+
+    Returns
+    -------
+    float
+        weighting of the features
+    """
+    if r <= (cutoff - width):
+        return 1.0
+    elif r >= cutoff:
+        return 0.0
+    else:
+        s = np.pi * (r - cutoff + width) / width
+        return 0.5 * (1.0 + np.cos(s))
+
+
+# %%
+# Lets plot the weighting for different widths.
+
+r = np.linspace(1e-3, 6.3, num=500)
+
+plt.plot([0, 6, 6, 6.3], [1, 1, 0, 0], c="k", label=r"Step function")
+
+for width in [6.0, 3.0, 1.0, 0.5, 0.1]:
+    weighting_values = [shifted_cosine(r=r_i, cutoff=6.0, width=width) for r_i in r]
+    plt.plot(r, weighting_values, label=f"Shifted cosine: $width={width}\,Å$")
+
+plt.legend()
+plt.xlabel(r"distance $r$ from the central atom $[Å]$")
+plt.ylabel("feature weighting")
+plt.show()
+
+# %%
+# From the plot we conclude that a larger ``width`` of the shifted cosine
+# function will decrease the feature values already
+# for smaller distances ``r`` from the central atom.
+
+# %%
+# Choosing the ``radial_scaling``
+# -------------------------------
+#
+# As mentioned above all atoms within the cutoff are taken into equally for
+# a descriptor. Since this might limit a ML model accrucary for atomistic systems
+# on top of a ``cutoff_function`` it is also useful to weight atoms
+# that are further away from the central atom with a lesser weighting
+# compared to atoms that are
+# closer. This can be achieved by a ``radial_scaling`` function
+# with a long-range algebraic decay and smooth behavior at :math:`r \rightarrow 0`
+#
+# .. math::
+#             u(r) = \begin{cases}
+#             \frac{1}{(r/r_0)^m} &\text{if c=0,}\\
+#             1 &\text{if m=0,} \\
+#             \frac{c}{c+(r/r_0)^m} &\text{else},
+#             \end{cases}
+#
+# where :math:`c` is the `rate`, :math:`r_0` is the `scale` parameter and :math`m`
+# the `exponent` of the RadialScaling function.
+#
+# The following functions computes such a radial scaling.
+
+
+def radial_scaling(r, rate, scale, exponent):
+    """Radial scaling function.
+
+    Parameters
+    ----------
+    r : float
+        distance between neighbouring atoms in Å
+    rate : float
+        decay rate of the scaling
+    scale : float
+        scaling of distance between atoms in Å
+    exponent : float
+        exponent of the decay
+
+    Returns
+    -------
+    float
+        weighting of the features
+    """
+    if rate == 0:
+        return 1 / (r / scale) ** exponent
+    if exponent == 0:
+        return 1
+    else:
+        return rate / (rate + (r / scale) ** exponent)
+
+
+# %%
+# In the following we show three different radial scaling functions, where the
+# first one are our default values.
```

Comment:
I'm not fond of "default values" here. There are no default values for a dataset, it depends on the task at hand.

Reply:
Yes, I agree. Even though we should give new users something in hand that they remember. Nothing is more confusing than always saying here a bunch of parameters but do not use them, because they might be wrong.